docs/conceptual/cryptography/session-keys.md

```diff
@@ -4,25 +4,34 @@ title: Session Keys
 
 Session keys are used by validators to sign consensus-related messages. `SessionKeys` is a generic, indexable type that is made concrete in the runtime.
 
-To create a Session key, validator operators must attest that a key acts on behalf of their Stash account (stake) and nominators. To do so, they create a certificate by signing the key with their Controller key. Then, they inform the chain that this key represents their Controller key by publishing the Session certificate in a transaction on the chain.
+You can declare any number Session keys. For example, the default Substrate node uses four. Other chains could have more or fewer depending on what operations the chain expects its validators to perform.
 
-You can declare any number Session keys. For example, the default Substrate node uses three. Other chains could have more or fewer depending on what operations the chain expects its validators to perform.
+In practice, validators amalgamate all of the session public keys into a certificate by signing the set of public keys with a "Controller" account and submitting a transaction. This on-chain certificate links a validator node with an account that holds funds. As such, that account can be credited with rewards or slashed based on the node's behavior.
+
+The runtime declares what session keys will be included in a certificate (`runtime/src/lib.rs`):
 
 ```rust
 pub struct SessionKeys {
```

Comment:
This needs to be wrapped into [impl_opaque_keys!](https://github.com/paritytech/substrate/blob/master/primitives/runtime/src/traits.rs#L1142)